static/scss/partials/onboarding.scss

```diff
@@ -0,0 +1,68 @@
+// Protocol
+@import "libs/protocol/css/templates/multi-column.scss";
+
+.c-onboarding {
+    border-radius: $border-radius-md;
+    border: 1px solid $color-light-gray-40;
+    padding: $layout-xs;
+    margin-bottom: $spacing-lg;
+
+    h2 {
+        @include text-title-2xs;
+        color: $color-black;
+        margin-bottom: $spacing-lg;
+    }
+    
+}
+
+.c-onboarding-steps {
+    margin: 0;
+    padding: 0;
+    gap: $spacing-lg;
+}
+
+.c-onboarding-step {
+    background-color: $color-white;
+    border-radius: $border-radius-md;
+    padding: $layout-xs;
```

Comment:
Should this padding be `$spacing-lg` instead? Both `$layout-xs` and `$spacing-lg` are 24px.

Reply:
Probably. There are no hard rules on when to use which token, and there's some overlap in the middle which makes it even more confusing. 

The general idea is that "spacing" tokens are for small spaces inside components and "layout" tokens are for big spaces between components, but we've never really spelled that out in writing and probably haven't even followed it consistently within Protocol.

Since this is padding inside a component I'd recommend `$spacing-lg`.